tests/test_login.py

```diff
@@ -5,34 +5,35 @@
 
 import runhouse as rh
 import sky
-from runhouse.globals import configs
+from runhouse.rns.login import _login_download_secrets
 
 
 def add_secrets_to_vault(headers):
     """Add some test secrets to Vault"""
     # Add real credentials for AWS and SKY to test sky status
-    rh.Secrets.put(
+    rh.provider_secret(
+        name="/aws",  # add backslash / to name to force it to be vault secret
         provider="aws",
-        secret={
+        values={
             "access_key": os.getenv("TEST_AWS_ACCESS_KEY"),
             "secret_key": os.getenv("TEST_AWS_SECRET_KEY"),
         },
-        headers=headers,
-    )
-    rh.Secrets.put(
+    ).save(headers=headers)
+
+    rh.provider_secret(
+        name="/sky",
         provider="sky",
-        secret={
-            "ssh_private_key": os.getenv("TEST_SKY_PRIVATE_KEY"),
-            "ssh_public_key": os.getenv("TEST_SKY_PUBLIC_KEY"),
+        values={
+            "private_key": os.getenv("TEST_SKY_PRIVATE_KEY"),
+            "public_key": os.getenv("TEST_SKY_PUBLIC_KEY"),
         },
-        headers=headers,
-    )
-    rh.Secrets.put(provider="snowflake", secret={"token": "ABCD1234"}, headers=headers)
-    rh.Secrets.put(
-        provider="ssh",
-        secret={"key-one": "12345", "key-one.pub": "ABCDE"},
-        headers=headers,
-    )
+    ).save(headers=headers)
+
+    rh.provider_secret(
+        name="/snowflake",
+        provider="snowflake",
+        values={"token": "ABCD1234"},
+    ).save(headers=headers)
 
 
 @pytest.mark.logintest
```

Comment:
At some point I'd love to move these tests inside a container so we can run them every time as normal unit tests without worrying about what's going on in the runner's filesystem.